content.php

```diff
@@ -11,6 +11,17 @@
 
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 
+	<?php
+	/**
+	 * primer_before_content hook.
+	 *
+	 * @hooked primer_woo_shop_message - 10
```

Comment:
@EvanHerman never mind, I just read your reasoning in #91 and it makes sense to me 👍